examples/01-basic/11-file-uploading-uppy/App.tsx

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import "@blocknote/core/fonts/inter.css";
+import { BlockNoteView } from "@blocknote/mantine";
+import "@blocknote/mantine/style.css";
+import { useCreateBlockNote, FilePanelController } from "@blocknote/react";
+
+import Uppy from "@uppy/core";
+import XHR from "@uppy/xhr-upload";
+import { Dashboard } from "@uppy/react";
+
+import "@uppy/core/dist/style.min.css";
+import "@uppy/dashboard/dist/style.min.css";
+
+export default function App() {
+  // Creates a new editor instance.
+  const editor = useCreateBlockNote({
+    initialContent: [
+      {
+        type: "paragraph",
+        content: "Welcome to this demo!",
+      },
+      {
+        type: "paragraph",
+        content: "Upload an image using the button below",
+      },
+      {
+        type: "image",
+      },
+    ],
+  });
+
+  const uppy = new Uppy().use(XHR, {
+    endpoint: "https://tmpfiles.org/api/v1/upload",
+  });
+
+  uppy.on("upload-success", (file: any, response: any) => {
+    // File should be removed from the uppy instance after successful upload.
+    if (response.status === 200) {
+      uppy.removeFile(file.id);
+    }
+
+    const block = editor.document[2];
```

Comment:
why editor.document[2]? This seems hardcoded?

Reply:
Manipulating blocks example also have hard coded values for updating a block

Reply:
should be fixed now